penguin/bravo.c

```diff
@@ -5,7 +5,7 @@
  * to something other than zero.
  */
 
-int penguin_bravo(void) {
+short penguin_bravo(void) {
    return 0;
```

Comment:
I thought the plan was to change the return value here as well? But I guess the order doesn't matter, just keep it in mind for a later PR.

Reply:
Yes, thanks for reminding me, there is another PR open for this.